modules/virt-hot-unplugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-unplugging-a-virtual-disk-using-the-web-console{context}"]
+= Hot-unplugging a virtual disk using the web console
+
+Hot-unplug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* Your virtual machine must be running.
+* You must have at least one data volume or persistent volume claim (PVC) available and hot-plugged.
+
+.Procedure
+
+. Click *Workloads → Virtualization → Virtual Machines* from the side menu.
+
+. Select the running virtual machine with the disk you want to hot-unplug.
+
+. Click *Actions* and select *Stop Virtual Machine*.
```

Comment:
I don't think you want to be stopping the VM, the whole point of hot unplugging is that the VM stays running while adding/removing the volumes.

Reply:
Agreed. @awels, I'm not very familiar with this feature, but I assume hot-unplugging is just deleting the hot-pluggable disk while the VM is running, correct?

If that's the case, then this step should be:

In the *Disks* tab, click the kebab menu of the hot-pluggable disk you want to hot-unplug and click *Delete*.